### PR TITLE
Update the main repository

### PR DESCRIPTION
**Pull request by zz9pzza**
General catch up so the config doesn't just live in my repository.

### CODE REVIEW

File: config/email_templates/2014-10/email.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                        <table width="100%" border="0" cellspacing="0" cellpadding="0">
+
+
+                          <tr>
+                            <td align="left" valign="top">
+                              <div style="font-family:Lucida Grande, Verdana, Geneva, sans-serif; font-size:1em; line-height:1.5em; color:#000000">
+
+<!-- !message_body-->
+<p>
+Dear [% username %],
```
Comment (sarken) [inline config/email_templates/2014-10/email.html]: This is missing a closing </p>

File: config/email_templates/2014-10/email.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+                        </table>
+                      </td>
+                    </tr>
+                    
+                    <tr>
+                      <td align="left" valign="top" style="padding:15px 25px 25px;" bgcolor="#990000">
+                        <div style="font-family:Lucida Grande, Verdana, Geneva, sans-serif; font-size:.7em; line-height:1.25em; color:#ffffff;">
+<!-- !footer -->
+<p>Please note: if you don’t want to receive e-mails like this one, you can set your <a href="http://archiveofourown.org/users/[%username_url%]/preferences" style="color:#ffffff;text-decoration:underline" >preferences</a> at the Archive not to receive admin messages.</p>
```
Comment (sarken) [inline config/email_templates/2014-10/email.html]: We don't want to include this.

File: config/email_templates/2015-05/email.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                        <table width="100%" border="0" cellspacing="0" cellpadding="0">
+
+
+                          <tr>
+                            <td align="left" valign="top">
+                              <div style="font-family:Lucida Grande, Verdana, Geneva, sans-serif; font-size:1em; line-height:1.5em; color:#000000">
+
+<!-- !message_body-->
+<p>
+Dear [%username%,
```
Comment (sarken) [inline config/email_templates/2015-05/email.html]: Missing a closing ] and a closing </p>

File: config/email_templates/2014-10/email.html (hunk 1 above)
Comment (zz9pzza) [inline config/email_templates/2014-10/email.html]: This is historical, I am pretty sure it is what we sent out, or the nearest i have to that.

File: config/email_templates/2016-04/email.text [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Dear [% username %],
+
+The Archive of Our Own relies entirely on donations from you ( http://bit.ly/A16members ) donations from you and volunteer work. The AO3 achieved an incredible milestone recently: two million fanworks! We continue to grow each day, thanks to the support of fans everywhere.
```
Comment (sarken) [inline config/email_templates/2016-04/email.text]: This says "donations from you" twice -- once before the link and once after. We should get rid of the one after.

File: config/email_templates/2016-04/email.text [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Your donations help keep the Archive running, along with other Organization for Transformative Works (OTW) projects like Fanlore, Legal Advocacy, and our peer-reviewed academic journal, Transformative Works and Cultures. 
+
+A small donation can make a big difference. If you donate US$10 or more, you can choose to become a member of the OTW. If you donate US$50 or more, you can also opt to receive a thank you gift. If you are able to do so, please donate today! ( http://bit.ly/A16members )
```
Comment (sarken) [inline config/email_templates/2016-04/email.text]: I'd change the end of this to simply `If you are able to do so, please donate today: http://bit.ly/A16members`

File: config/email_templates/2016-04/email.text [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+If you have already donated during this drive, thank you! Your support means a lot. If you have questions about donating, please contact the OTW Development and Membership committee ( http://transformativeworks.org/contact_us/contact-development-and-membership/ ).
```
Comment (sarken) [inline config/email_templates/2016-04/email.text]: The end of this can be simplified to: `If you have questions about donating, please contact the OTW Development and Membership committee: http://transformativeworks.org/contact_us/contact-development-and-membership/`

File: scripts/codeship/run_codeship_tests.sh [hunk 7]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -x
```
Comment (sarken) [inline scripts/codeship/run_codeship_tests.sh]: Is this supposed to be in here? It's the only thing that doesn't seem related to email.

Comment (sarken) [inline scripts/codeship/run_codeship_tests.sh]: The two Codeship things, I mean, not just this file.

Comment (zz9pzza) [inline scripts/codeship/run_codeship_tests.sh]: Yes, it was going to move over here but I am happy it staying in the other repository. I am happy to remove these two files.

File: config/email_templates/2016-04/email.html [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+        </tr>
+        <tr>
+          <td align="left" valign="top">
+            <table width="100%" border="0" cellspacing="0" cellpadding="0">
+              <tr>
+                <td width="354" align="center" valign="middle" bgcolor="#990000">
+                  <div style="font-family:Georgia, Times New Roman, Times, serif; font-size:28px; line-height:1em; color:#ffffff; font-weight:bold;">
+
+<!-- !banner -->
+                    <a href="http://archiveofourown.org/" style="color:#ffffff; text-decoration:none"><span style="color:#ffffff; text-decoration:none">Archive of Our Own</span></a>
```
Comment (sarken) [inline config/email_templates/2016-04/email.html]: Would we like to change this to https?

File: config/email_templates/2016-04/email.html [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+
+                          <tr>
+                            <td align="left" valign="top">
+                              <div style="font-family:Lucida Grande, Verdana, Geneva, sans-serif; font-size:1em; line-height:1.5em; color:#000000">
+
+                              <!-- !message_body-->
+<p>Dear [% username %],</p>
+
+<p>
+The Archive of Our Own relies entirely on <a href="http://bit.ly/A16members" style="color:#990000; text-decoration:underline" >donations from you</a> and volunteer work. The AO3 achieved an incredible milestone recently: two million fanworks! We continue to grow each day, thanks to the support of fans everywhere.
```
Comment (sarken) [inline config/email_templates/2016-04/email.html]: There's a stray space in the HTML here: `text-decoration:underline" >`

Comment (zz9pzza) [inline config/email_templates/2016-04/email.html]: so it should be:

style="color:#990000;text-decoration:underline"

File: config/email_templates/2016-04/email.html [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+<p>
+Your donations help keep the Archive running, along with other Organization for Transformative Works (OTW) projects like Fanlore, Legal Advocacy, and our peer-reviewed academic journal, <cite>Transformative Works and Cultures</cite>. 
+</p>
+
+<p>
+A small donation can make a big difference. If you donate US$10 or more, you can choose to become a member of the OTW. If you donate US$50 or more, you can also opt to receive a thank you gift. If you are able to do so, please <a href="http://bit.ly/A16members" style="color:#990000; text-decoration:underline">donate today!</a>
+</p>
+
+<p>
+If you have already donated during this drive, thank you! Your support means a lot. If you have questions about donating, please <a href="http://transformativeworks.org/contact_us/contact-development-and-membership/" style="color:#990000; text-decoration:underline" >contact the OTW Development and Membership committee</a>.
```
Comment (sarken) [inline config/email_templates/2016-04/email.html]: Another stray space between the " and the > : `text-decoration:underline" >`

File: config/email_templates/2016-04/email.html (hunk 9 above)
Comment (sarken) [inline config/email_templates/2016-04/email.html]: No, the extra space is between the " and the > -- the space after the ; is good.

File: config/email_templates/2016-04/email.html (hunk 8 above)
Comment (zz9pzza) [inline config/email_templates/2016-04/email.html]: I wouldn't right now, it is just a link to the archive. I would prefer we got the  rid of the absolute paths before we actively use https.

Comment (sarken) [inline config/email_templates/2016-04/email.html]: Okay, sounds good!
